src/window.rs

```diff
@@ -401,6 +403,26 @@ impl WindowBuilder {
         self
     }
 
+    /// Build window with parent window.
+    ///
+    /// The default is `None`.
+    ///
+    /// ## Safety
+    ///
+    /// `parent_window` must be a valid window handle.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Windows** : A child window has the WS_CHILD style and is confined
+    /// to the client area of its parent window. For more information, see
```

Comment:
On X11 the child is also confined to the parent window's area (at least when I last tested).